velox/exec/tests/utils/QueryAssertions.cpp

```diff
@@ -772,11 +1110,21 @@ static bool compareOrderedPartitions(
       return false;
     }
   }
-  if (not compareMaterializedRows(left.second, right.second)) {
-    return false;
-  }
 
-  return true;
+  auto width = left.second.begin()->size();
+  auto [floatColumnIndices, nonFloatColumnIndices] =
+      getFloatColumnIndices(left.second, width);
+  MaterializedRowsForEpsilonComparison rowsForComparison{
+      floatColumnIndices, nonFloatColumnIndices};
+  if (!floatColumnIndices.empty() &&
+      rowsForComparison.prepareForEpsilonComparison(
+          left.second, right.second)) {
+    return rowsForComparison.resultsMatch();
+  }
+  // Compare the results directly without epsilon. This may cause false alarm
```

Comment:
Consider adding a note similar to kDuckDbTimestampWarning to the error message in case there are floating-point columns.

Reply:
> Consider adding a note similar to kDuckDbTimestampWarning to the error message in case there are floating-point columns.

Curious what are your thoughts on this.

Reply:
> > Consider adding a note similar to kDuckDbTimestampWarning to the error message in case there are floating-point columns.
> 
> Curious what are your thoughts on this.
Sorry, I missed this. I'll add a note.